buzzmobile/launch/params.launch

```diff
@@ -0,0 +1,5 @@
+<launch>
+    <group ns="buzzmobile">
+        <rosparam command="load" file="$(find buzzmobile)/constants.yaml"/>
+    </group>
```

Comment:
So the way our main launch files work is they currently have things like:

```
<third party nodes>
<buzzmobile namespace>
    <rosparam constants>
    <include our nodes>
</buzzmobile namespace>
```

I suggest you remove the buzzmobile namespace from this `params.launch`, put it in `buzzmobile/launch/includes/`, modify `main.launch` (and other main files under `buzzmobile/launch/temp` to be like:

```
<third party nodes>
<buzzmobile namespace>
    <include params>
    <include our nodes>
</buzzmobile namespace>
```

and finally create test-specific launch files (somewhere like `buzzmobile/tests/launch/parms_test.launch`), which define a namespace and include the params, like such:

```
<buzzmobile namespace>
    <include params>
</buzzmobile namespace>
```

This way we can be consistent all around, reuse as much as possible, and still be able to define test-specific launch files.